src/chromatix/utils/initializers.py

```diff
@@ -143,6 +143,7 @@ def zernike_aberrations(
     NA: float,
     ansi_indices: Sequence[int],
     coefficients: Sequence[float],
+    normalization: bool = True,
```

Comment:
I will probably rename this to normalize, but other than that looks good and I'll merge it into hackathon_2024 first!